pyfr/backends/cuda/kernels/base.mako

```diff
@@ -6,6 +6,7 @@
 
 // Typedefs
 typedef ${pyfr.npdtype_to_ctype(fpdtype)} fpdtype_t;
+typedef unsigned int uint32_t;
```

Comment:
Is this needed for HIP, too?

Reply:
I'm getting the AMD machine I have setup - will test and report back.

Reply:
Any updates here?

Reply:
Just following up here on the AMD side.